deploy/README.md

```diff
@@ -0,0 +1,50 @@
+# Deploy
+## Run with Docker Compose
+Running the Serai infrastructure is easy with Docker.
+
+We utilize compose profiles to easily orchestrate various pieces of the infrastructure.
+
+**Example:** `docker compose --profile cluster-coins-sm up`
+
+All commands are assumed to be ran from the deploy folder, not the serai root folder.
+
+### Profiles:
+* base - single node, named base
+* coins - node clients for coins only (XMR, BTC, ETH)
```

Comment:
Sort BTC, ETH, XMR please. I'd also prefer this work incrementally, so coins, base, cluster, cluster-coins, yet you let me know if you prefer the various coins be buried as if they're of little importance.

Reply:
updated, like this?